_posts/pl/2021-12-XX-transactions.md

```diff
@@ -0,0 +1,217 @@
+---
+layout:    post
+title:     "Transakcje - co warto wiedzieć?"
+date:      2021-12-XX 08:00:00 +0100
+published: false
+didyouknow: false
+lang: pl
+author:    rmastalerek
+image:     /assets/img/posts/2021-12-14-transactions/transactions.jpg
+tags:
+    - transactions
+    - spring
+    - java
+    - transactional
+---
+
+Czy ktokolwiek, po zamówieniu w restauracji kawy i ciastka chciałby zapłacić za oba, a otrzymać tylko czarny, gorący napój? Czy do zaakceptowania byłby fakt, że po wysłaniu przelewu na pokaźną kwotę, uszczuplony zostałby jedynie nasz rachunek, a kwota nie zasiliłaby konta odbiorcy? Podane przykłady mają na celu lepsze zilustrowanie zagadnienia, które zostanie poruszone w niniejszym wpisie. Pewne rzeczy mają sens tylko wtedy, gdy wykonywane są kompleksowo. Tym razem chciałbym przedstawić krótkie wprowadzanie do transakcji. Wpis zostanie podzielony na 2 części. W części pierwszej skupię się na przybliżeniu samej definicji oraz zasady działania transakcyjności w systemach baz danych oraz wykorzystaniu adnotacji _@Transactional_ w Springu. Kolejna część poświęcona będzie problemom jakie napotkaliśmy w codziennej pracy, co pozwoliło je zidentyfikować oraz jak zostały wyeliminowane.
+
+# Transakcja
```

Comment:
Nie stosujemy nagłówków pierwszego stopnia. Taki nagłówek powinien być tylko jeden i jest tworzony z automatu przez jekylla (zawiera tytuł posta). Zmień na nagłówek drugiego stopnia

Reply:
jesteś pewien? masa postóœ ma nagłówki pierwszego poziomu, choćby ostatni z 11-23 o json i jsonb?

Reply:
widzę też od groma wpisów zaczynających od ### czy #### :facepalm: rozumiem, że poleciane tylko "na oko" bez pomyślenia o strukturze.

Reply:
To ja się w międzyczasie przejdę po wszystkich artykułach i pozamieniam h2 na h1, bo nie zwracałem na to uwagi, a z perspektywy SEO jest to dosyć istotne z tego co zrozumiałem.

Reply:
ja zrozumiałem, że na odwrót. h1 na h2, bo h1 to tytuł wpisu :)

Reply:
tru, typo :D

Reply:
zmieniłe mnagłówki, odpowiednio:
# -> ##
## -> ###